### ISSUE TITLE
App shows "Connected Trezor is used by another application”, even though there was not using any other Suite instance at that time

### ISSUE THREAD

**Issue by mnuky** [trezor/trezor-suite#10610]
It happened that app shows that the Trezor is used by another application - it happened to internal user when he was using the app but it may also happen when user unplug the device. 
![Image](https://github.com/trezor/trezor-suite/assets/84183817/e64baff5-bc3e-4955-baaa-fae91ecac7a5)

Please replicate the bug

**Comment by mroz22** [comment]
let me guess, did it happen after app was minimized/put to background? 
if yes, is there a way how to do some work before this happens? @Nodonisko ?

**Comment by Nodonisko** [comment]
Yes, we can do something before app goes background. But if there is something running like discovery we can't gurantee it will get to device in time.

**Comment by mroz22** [comment]
if what I think is really the reason, we have 2 options:
1. catch some "before goes to sleep" event and fire release session.
2. just ignore sessions completelly.

**Comment by PeKne** [comment]
@Nodonisko are you working on that?

**Comment by mroz22** [comment]
Guys, could be it won't be so simple. App going to sleep could cause further problems not only this. especially, if it happens during communication with trezor. You could easily end up with desynchronization of communication with device meaning that you could receive unexpected responses from previous calls 🤔

**Comment by Hannsek** [comment]
Could it be fixed by the new protocol?

**Comment by Nodonisko** [comment]
Also don't forget about edge case when user may use Trezor in browser on Android. Not sure how it will behave then.

@PeKne I don't work on this, just trying to figure out best solution.

**Comment by mnuky** [comment]
@Nodonisko that was the reason why this message was created - to handle the situation, when Trezor is used in Android browser: https://github.com/trezor/trezor-suite/issues/10104
